pallets/ds-accounts/src/lib.rs

```diff
@@ -82,7 +87,31 @@ impl<
     }
 }
 
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+#[derive(Encode, Decode, Clone, Default)]
+pub struct UAVStruct<SerialNumber, MetaIPFS, OwnerId> { 
+    pub uav_id: SerialNumber,
+    pub metadata_ipfs_hash: MetaIPFS,
+    pub managed_by: OwnerId,
+}
+
+impl<
+    SerialNumber: Default + Copy,
+    MetaIPFS: Default + Copy,
+    OwnerId: Parameter + Member + MaybeSerializeDeserialize + Ord + Default,
+    > UAVStruct<SerialNumber, MetaIPFS, OwnerId>
+    { 
+        pub fn new_default() -> Self {
```

Comment:
default value should be implemented via Default trait.

Reply:
removed "new_default", added "new" function. 
Works differently. Takes required for UAVStruct fields as arguments(who, meta etc), returns UAVStruct.